docs/source/index.rst

```diff
@@ -51,11 +55,16 @@ Code reference
 .. toctree::
    :maxdepth: 1
 
+   contributing
+
    basic
    attempt
    cli
    command
+   exception
    interactive
+   _config
+   _plugins
```

Comment:
Exposing these classes here suggests to me that their interfaces are consumable by custom plugin developers, is there some way we can tag these with a note that these are internal only classes with no guarantee of a stable API?

Reply:
added in rst file for each of these classes